definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -705,5 +705,47 @@
     "CaseFieldID": "LabelContinueBecauseOfDispensedAccepted",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/09/2020",
```

Comment:
is this so old? :)

Reply:
@yarpo Would go though LiveFrom date's and update, somehow it always skips me :)